libkbfs/block_retrieval_worker_test.go

```diff
@@ -263,13 +263,26 @@ func TestBlockRetrievalWorkerShutdown(t *testing.T) {
 	shutdown := false
 	select {
 	case <-ch:
+		t.Fatal("Expected not to retrieve a result from the Request.")
 	case continueCh <- nil:
+		t.Fatal("Expected the block getter not to be receiving.")
 	default:
 		shutdown = true
 	}
 	require.True(t, shutdown)
-	w.Shutdown()
-	require.True(t, shutdown)
+
+	// Ensure the test completes in a reasonable time.
+	timer := time.NewTimer(10 * time.Second)
+	doneCh := make(chan struct{})
+	go func() {
+		w.Shutdown()
```

Comment:
Is `w.Shutdown()` synchronous? https://github.com/keybase/kbfs/pull/1030/files#diff-e3ef24ce4d67246c2920d7d276db5648R264 seems to suggest it's async?